packages/react-codemirror/src/e2e_tests/configuration.spec.tsx

```diff
@@ -95,3 +95,25 @@ test('can set/unset onFocus/onBlur', async ({ mount, page }) => {
     expect(blurFireCount).toBe(1);
   }).toPass();
 });
+
+test('aria-label is not set by default', async ({ mount, page }) => {
+  await mount(<CypherEditor />);
+
+  const textField = page.getByRole('textbox');
+
+  await expect(async () => {
+    expect(await textField.getAttribute('aria-label')).toBeNull();
+  }).toPass();
```

Comment:
I think you don't need the double wrap in an expect and this would do:

```
  expect(await textField.getAttribute('aria-label')).toBeNull();
```

Reply:
yeh fair comment - I've removed the double wrapped assertion